### ISSUE TITLE
size mismatch for prediction_layers.0.conf_layer.weight

### ISSUE THREAD

**Issue by pzs7602** [dbolya/yolact#387]
I want to train a model on my own datatset, the train command is:
python train.py --config=yolact_scratch_config --resume=weights/yolact_im700_54_800000.pth --start_iter=0
the train can begin but error message displayed on console:

Resuming training, loading weights/yolact_im700_54_800000.pth...
Ignoring "Error(s) in loading state_dict for Yolact:
	size mismatch for prediction_layers.0.conf_layer.weight: copying a param with shape torch.Size([243, 256, 3, 3]) from checkpoint, the shape in current model is torch.Size([6, 256, 3, 3]).
	size mismatch for prediction_layers.0.conf_layer.bias: copying a param with shape torch.Size([243]) from checkpoint, the shape in current model is torch.Size([6]).
	size mismatch for semantic_seg_conv.weight: copying a param with shape torch.Size([80, 256, 1, 1]) from checkpoint, the shape in current model is torch.Size([1, 256, 1, 1]).
	size mismatch for semantic_seg_conv.bias: copying a param with shape torch.Size([80]) from checkpoint, the shape in current model is torch.Size([1])."

how to solve this? thanks in advance!

my config is:
dataset  =  <data.config.Config object at 0x0000023BE9DA4348>
num_classes  =  2
max_iter  =  800000
max_num_detections  =  100
lr  =  0.001
momentum  =  0.9
decay  =  0.0005
gamma  =  0.1
lr_steps  =  (280000, 600000, 700000, 750000)
lr_warmup_init  =  0.0001
lr_warmup_until  =  500
conf_alpha  =  1
bbox_alpha  =  1.5
mask_alpha  =  6.125
eval_mask_branch  =  True
nms_top_k  =  200
nms_conf_thresh  =  0.05
nms_thresh  =  0.5
mask_type  =  1
mask_size  =  16
masks_to_train  =  100
mask_proto_src  =  0
mask_proto_net  =  [(256, 3, {'padding': 1}), (256, 3, {'padding': 1}), (256, 3, {'padding': 1}), (None, -2, {}), (256, 3, {'padding': 1}), (32, 1, {})]
mask_proto_bias  =  False
mask_proto_prototype_activation  =  <function <lambda> at 0x0000023BE9DA7DC8>
mask_proto_mask_activation  =  <built-in method sigmoid of type object at 0x00007FFD2A09E910>
mask_proto_coeff_activation  =  <built-in method tanh of type object at 0x00007FFD2A09E910>
mask_proto_crop  =  True
mask_proto_crop_expand  =  0
mask_proto_loss  =  None
mask_proto_binarize_downsampled_gt  =  True
mask_proto_normalize_mask_loss_by_sqrt_area  =  False
mask_proto_reweight_mask_loss  =  False
mask_proto_grid_file  =  data/grid.npy
mask_proto_use_grid  =  False
mask_proto_coeff_gate  =  False
mask_proto_prototypes_as_features  =  False
mask_proto_prototypes_as_features_no_grad  =  False
mask_proto_remove_empty_masks  =  False
mask_proto_reweight_coeff  =  1
mask_proto_coeff_diversity_loss  =  False
mask_proto_coeff_diversity_alpha  =  1
mask_proto_normalize_emulate_roi_pooling  =  True
mask_proto_double_loss  =  False
mask_proto_double_loss_alpha  =  1
mask_proto_split_prototypes_by_head  =  False
mask_proto_crop_with_pred_box  =  False
augment_photometric_distort  =  True
augment_expand  =  True
augment_random_sample_crop  =  True
augment_random_mirror  =  True
augment_random_flip  =  False
augment_random_rot90  =  False
discard_box_width  =  0.007272727272727273
discard_box_height  =  0.007272727272727273
freeze_bn  =  False
fpn  =  <data.config.Config object at 0x0000023B8DBF16C8>
share_prediction_module  =  True
ohem_use_most_confident  =  False
use_focal_loss  =  False
focal_loss_alpha  =  0.25
focal_loss_gamma  =  2
focal_loss_init_pi  =  0.01
use_class_balanced_conf  =  False
use_sigmoid_focal_loss  =  False
use_objectness_score  =  False
use_class_existence_loss  =  False
class_existence_alpha  =  1
use_semantic_segmentation_loss  =  True
semantic_segmentation_alpha  =  1
use_mask_scoring  =  False
mask_scoring_alpha  =  1
use_change_matching  =  False
extra_head_net  =  [(256, 3, {'padding': 1})]
head_layer_params  =  {'kernel_size': 3, 'padding': 1}
extra_layers  =  (0, 0, 0)
positive_iou_threshold  =  0.5
negative_iou_threshold  =  0.4
ohem_negpos_ratio  =  3
crowd_iou_threshold  =  0.7
mask_dim  =  32
max_size  =  550
force_cpu_nms  =  True
use_coeff_nms  =  False
use_instance_coeff  =  False
num_instance_coeffs  =  64
train_masks  =  True
train_boxes  =  True
use_gt_bboxes  =  False
preserve_aspect_ratio  =  False
use_prediction_module  =  False
use_yolo_regressors  =  False
use_prediction_matching  =  False
delayed_settings  =  []
no_jit  =  False
backbone  =  <data.config.Config object at 0x0000023B8DBF1688>
name  =  yolact_scratch
use_maskiou  =  False
maskiou_net  =  []
discard_mask_area  =  -1
maskiou_alpha  =  1.0
rescore_mask  =  False
rescore_bbox  =  False
maskious_to_train  =  -1
num_heads  =  5
epoch_size= 89
num_epoch= 8989

**Comment by jasonkena** [comment]
Just follow the instructions at https://github.com/dbolya/yolact/issues/36, it will ignore all these size mismatches.

**Comment by dbolya** [comment]
Yup, those size mismatches you can ignore with that issue.
